tapdb/sqlutils_test.go

```diff
@@ -268,3 +272,17 @@ func NewDbHandle(t *testing.T) *DbHandler {
 		DirectQuery:             db,
 	}
 }
+
+// NewDbHandleFromPath creates a new database store handle given a database file
+// path.
+func NewDbHandleFromPath(t *testing.T, dbPath string) *DbHandler {
```

Comment:
I'm a bit confused by the naming convention used here. If something is a handler, why would the variable names and functions to create such a handler only be referred to as "handle"? Or is that just a very consistent typo?
I still think "harness" is a better name here, since that's a term we very often use in all our projects for things that just hold test instances together and provide helper methods...

Reply:
The way I think about it is that the type struct is a handler and an instance of that type is a handle.

We can go with harness, I don't mind either way.

I'll put up another PR to rename if you don't mind.

Reply:
Okay, thanks for clarifying.
It would be nice to rename, but let's not make it a priority. Perhaps keep it in mind for the next time we add or change something to that struct?